src/vg-tooltip.js

```diff
@@ -289,12 +289,21 @@
     // the supplemented field option
     var supplementedFieldOption = {};
 
-    // supplement field name with underscore prefixes and suffixes to match the field names in item.datum
-    // for aggregation and timeUnit, this will add prefix "mean_", "yearmonth_"
+    // supplement a user-provided field name with underscore prefixes and suffixes to
+    // match the field names in item.datum
+    // for aggregation, this will add prefix "mean_" etc.
+    // for timeUnit, this will add prefix "yearmonth_" etc.
     // for bin, this will add prefix "bin_" and suffix "_start". Later we will replace "_start" with "_range".
     supplementedFieldOption.field = fieldDef.field ?
       vl.fieldDef.field(fieldDef) : fieldOption.field;
 
+    // if a temporal fieldDef has timeUnit, we store the unprefixed field name in options
+    // because we need the unprefixed field name in function removeDuplicateTimeFields()
+    // user should never have to provide unprefixedFieldName in options
+    if (fieldDef.type === "temporal" && fieldDef.timeUnit) {
+      supplementedFieldOption.unprefixedFieldName = fieldDef.field;
```

Comment:
Corner case:  What if you can two fieldDefs that use a temporal field T1 (one with timeUnit and one without), you should keep T1.  

That said, this wouldn't happen in our user study, so maybe file an issue that you can deal with later.